### DISCUSSION TITLE
MIRYOKU_NAV=VI does not shift top row left as stated in documentation

### DISCUSSION THREAD

**Discussion by UncleCJ** [manna-harbour/miryoku#141] [Q&A] [answered]
Copying https://github.com/UncleCJ/miryoku/commit/d0fcdac3dfd5e7959af0e04296bef067700e76d3 (in case the commit changes):

> Alter top row in MIRYOKU_NAV=VI to how it currently is (but should not be?)
> 
> [docs/reference/readme.org#vi-style](https://github.com/manna-harbour/miryoku/blob/master/docs/reference/readme.org#vi-style) states: "Columns are moved left one position to match ~hjkl~ positions, and the inner index column is moved to the pinkie column.  All same hand sub layers are modified in the same way" whereas [miryoku_layer_alternatives.h#L272](https://github.com/manna-harbour/miryoku_qmk/blob/miryoku/users/manna-harbour_miryoku/miryoku_babel/miryoku_layer_alternatives.h#L272) actually is `U_RDO, U_PST, U_CPY, U_CUT, U_UND`, same as the other alternatives.
> 
> Just alter the layout illustrations for now, until it can be properly understood and perhaps changed in implementation.
> 
> Thanks to [Kamal Al-Wendawi](https://www.linkedin.com/in/kamal-al-wendawi-236509158) for diligently helping me troubleshoot my layout illustrations

So yes, did I miss a change to this already, should it be fixed or add a description of the exception to documention?

**Comment by manna-harbour** [comment, marked as answer]
Thanks for the catch! The clipboard keys shouldn't change as they're not based on the nav positions, only the other 2 rows. So perhaps it should read "Other than the clipboard row, columns are moved...", or add "The clipboard row is unchanged." at the end

**Reply by crazzle** [reply]
Thanks for your answer. This info is still missing, can you add it? Thanks!

**Reply by manna-harbour** [reply]
Updated now.

The changed layers for vi-style Nav can be seen at https://github.com/manna-harbour/miryoku_babel#vi-style.
